frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepOrderPicker.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { StepOrderValue } from '~/types'
+import { Select } from 'antd'
+import { useActions, useValues } from 'kea'
+import { funnelLogic } from 'scenes/funnels/funnelLogic'
+import { ANTD_TOOLTIP_PLACEMENTS } from 'lib/utils'
+
+interface StepOption {
+    key?: string
+    label: string
+    value: StepOrderValue
+}
+
+const options: StepOption[] = [
+    {
+        label: 'Sequential',
+        value: StepOrderValue.ORDERED,
+    },
+    {
+        label: 'Strict Order',
+        value: StepOrderValue.STRICT,
+    },
+    {
+        label: 'Any Order',
+        value: StepOrderValue.UNORDERED,
+    },
+]
+
+export function FunnelStepOrderPicker(): JSX.Element {
+    const { filters } = useValues(funnelLogic)
+    const { setFilters } = useActions(funnelLogic)
+
+    return (
+        <Select
+            id="funnel-step-order-filter"
+            data-attr="funnel-step-order-filter"
+            defaultValue={StepOrderValue.ORDERED}
+            value={filters.funnel_order_type || StepOrderValue.ORDERED}
+            onSelect={(stepOrder) => setFilters({ funnel_order_type: stepOrder })}
+            listHeight={440}
+            dropdownMatchSelectWidth={false}
+            dropdownAlign={ANTD_TOOLTIP_PLACEMENTS.bottomRight}
+            optionLabelProp="label"
+        >
+            {options.map((option) => {
+                return (
+                    <Select.Option key={option.value} value={option.value} label={option.label}>
```

Comment:
Nit: I don't think you need the `label` & the `children`. Even more, as we don't do any special handling in the display options, you might be better off just using the `options` prop.

Reply:
> lgtm! Perhaps my only feedback (and sorry should've said it in the issue about the design) is that I think it's worth changing the numbers when doing any ordering, perhaps do letters instead? a symbol? or remove them altogether? I think it can help make clear the conversion sequence, particularly because visually steps do keep looking sequential.

Thanks for the review! That's also something that I suggested, but @clarkus brought up a good point [here](https://github.com/PostHog/posthog/issues/5552#issuecomment-899883857) about a couple of use cases where step identifiers would be nice to have for "any order". What do you think about this use case?

Reply:
What's the problem we're trying to solve with that proposed change? Is it that the "any order" evaluation mode is not obvious enough? I don't think letters are a better option than numbers. They both imply a sequencing or ordering, just based on different criteria.

Reply:
Right! So I think the problem is that any ordering can be confusing when visualizing the actual graph as you still see something that looks like a sequence of steps. Initially I was envisioning something like having asterisks instead of numbers so it's clear that this is not a sequence.